app/controllers/application_controller.rb

```diff
@@ -1,5 +1,7 @@
 class ApplicationController < ActionController::Base
+  include Pundit
   protect_from_forgery with: :exception
+  before_action :authenticate_user!
```

Comment:
Will this be a problem if user is only guest (not logged in)?

Reply:
This is temporary (until I start working on actual model for stories etc.), to demonstrate the functionality of redirecting & logging in/out